### DISCUSSION TITLE
Ability to manually ban certain nodes

### DISCUSSION THREAD

**Discussion by ApfelBirneKreis** [Chia-Network/chia-blockchain#17679] [Ideas]
Hello fellow farmers and devs,
ist there a way to manually ban a peer? preferably a cli command?

I have some peers that connect to me again as soon as i get rid of them manually. Their height is around 10000 blocks, not going up and not going down. Always the same IP and Node ID. I dont want them in my network since they are draining my upload. (its around 800 - 1200 MB daily towards those nodes).

If there is no such funktionality yet, can this be implemented?

**Comment by OverActiveBladderSystem** [comment]
> ist there a way to manually ban a peer? preferably a cli command?

Windows Firewall for example (link below) can be utilized via (an administrative) command prompt or powershell, I imagine a linux based system is able to easily do this as well. So there is a solution built into the OS if you really feel the need to do something via command line.

https://learn.microsoft.com/en-us/windows/security/operating-system-security/network-security/windows-firewall/configure-with-command-line?tabs=cmd

**Reply by BrandtH22** [reply]
Hey @ApfelBirneKreis , unfortunately there is currently no manual ban list that can be added to for this use case. 
Thank you for the recommendation and note that I have updated the tickets category and added some tags to identify it as a feature request for our engineers to review as time permits.

As OverActive noted, there are likely some OS tools that would allow for the banning/blocking of specific IPs which could be used as a workaround :+1:
